#+STARTUP: showall beamer
#+COLUMNS: %40ITEM %10BEAMER_env(Env) %9BEAMER_envargs(Env Args) %4BEAMER_col(Col) %10BEAMER_extra(Extra)
# +BEAMER_HEADER: \titlegraphic{\includegraphics[height=1.5cm]{InstLogo}}

#+TITLE: \small Microsoc Seminar Series 2018-09-04: \newline\LARGE Soil-persistent /E. coli/ and Mobile elements
# +SUBTITLE: Or, an overview of mobile elements in /E. coli/
#+AUTHOR: Nicholas Waters
# +DATE: \today
# #+SUBTITLE
#+INSTITUTE: National University of Ireland, Galway, Ireland \linebreak The James Hutton Institute, Dundee, Scotland}
#+LATEX_HEADER: \institute{National University of Ireland, Galway, Ireland \\ The James Hutton Institute, Dundee, Scotland}

#+OPTIONS: H:2 toc:1

#+LATEX_HEADER: \PassOptionsToPackage{draft}{graphicx}

# ################################################ Set the Aspect Ratio #############################
# +LaTeX_CLASS_OPTIONS: [17pt,aspectratio=169]
#+LaTeX_CLASS_OPTIONS: [15pt,aspectratio=43]

#+LATEX_HEADER: \renewcommand*\familydefault{\sfdefault} % make font sansserif
#+LATEX_HEADER: \newcommand{\bt}{\textasciigrave}
#+LATEX_HEADER: \usepackage{xcolor}
#+LATEX_HEADER: \def \ttilde {\raisebox{-.6ex}\textasciitilde~}
#+LATEX_HEADER: \setlength\parindent{0pt} %set indent to zero
#+LATEX_HEADER: \setlength{\parskip}{1em}
#+LATEX_HEADER: \definecolor{bg}{HTML}{B1F4A0}
# +LATEX_HEADER: \lstset{basicstyle=\linespread{1.1}\ttfamily\scriptsize, breaklines=true, backgroundcolor=\color{bashcodebg}, xleftmargin=0.5cm, language=bash, showstringspaces=false, columns=fullflexible}
#+LATEX_HEADER: \usepackage{tcolorbox}

#+LATEX_HEADER: \usepackage{etoolbox}
#+LATEX_HEADER: \usepackage{geometry}
#+LATEX_HEADER: \usepackage[colorlinks = true, linkcolor = blue, urlcolor  = blue, citecolor = blue, anchorcolor = blue]{hyperref}
#+LATEX_HEADER: \let\oldv\verbatim
#+LATEX_HEADER: \let\oldendv\endverbatim
#+LATEX_HEADER: \def\verbatim{\par\setbox0\vbox\bgroup\scriptsize\oldv}
#+LATEX_HEADER: \def\endverbatim{\oldendv\egroup\fboxsep0pt \noindent\colorbox[gray]{0.8}{\usebox0}\par}
#+LaTeX_HEADER: \usepackage{array, booktabs, xcolor, tikz}

#+LaTeX_HEADER: \usepackage{graphicx}

#+LaTeX_HEADER: \usetikzlibrary{arrows, calc, spy, shapes}
#+LaTeX_HEADER: \tikzstyle{square} = [draw]
#+LaTeX_HEADER:\addtobeamertemplate{footnote}{\tiny}{} %\vspace{2ex}}

# set light/ dark theme here
#+LaTeX_HEADER:\usetheme[style=dark]{NUIG}
# light
# +BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon.pdf}}
# dark
#+BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon_D2.pdf}}

* Background
** Project Overview
- /E. coli/ has been found to persist stably in the soil
- Isolates were cultured from lysimeter leachate
- Strains were sequenced, resulting in *149* soil-persistent /E. coli/ genome

** Research Questions
- What types of /E. coli/ are able to persist in soil?
#+BEAMER: \pause
- What virulence factors are harboured by these strains?
#+BEAMER: \pause
- *What can we infer about adaptation?*
#+BEAMER: \pause
- Can we differentiate soil-persistent /E. coli/ from recent contamination?

* /E. coli/ Pangenome
** Collection overview
# ** Average Nucleotide Identity
# http://widdowquinn.github.io/pyani/
# #+BEGIN_LaTeX
# \begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
#   \node[anchor=south west,inner sep=0] (image) at (0,0){
#   \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
# \end{tikzpicture}
# #+END_LaTeX

# ** Average Nucleotide Identity
# :PROPERTIES:
# :BEAMER_opt: fragile
# :END:
# http://widdowquinn.github.io/pyani/
# #+BEGIN_LaTeX
# \begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
#   \node[anchor=south west,inner sep=0] (image) at (0,0){
#   \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
#     \spy on ($.53*(image.north)+0.61*(image.east)$) in node at ([xshift=4cm]image.east);
# \end{tikzpicture}
# #+END_LaTeX


** Annotation
#+BEGIN_LaTeX
\noindent
\begin{tikzpicture}
\node [anchor=west] (note) at (-1,4) {\Large Partial};
\begin{scope}[xshift=1.5cm]
    \node[anchor=south west,inner sep=0] (image) at (0,0) {\includegraphics[width=0.67\textwidth]{./frequentFigs/weird_gene3.png}};
    \begin{scope}[x={(image.south east)},y={(image.north west)}]
        \draw[red,ultra thick,rounded corners] (0.5,0.05) rectangle (0.55,0.15);
        \draw [-latex, ultra thick, red] (note) to (0.48,0.1);
    \end{scope}
\end{scope}
\end{tikzpicture}%

#+END_LaTeX

** Annotation Correction with annofilt performance
#+LaTeX: \vskip -2ex
#+LaTeX: \url{https://nickp60.github.io/annofilt/}\vspace{2em}
#+LaTeX: \vskip -6ex
*** Col left 							      :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.4
   :BEAMER_opt: [t]
   :END:
#+CAPTION:
#+NAME:   fig:Annofilt
#+ATTR_LATEX: :width \textwidth
[[file:./frequentFigs/ent2d.png]]

*** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.4
   :BEAMER_opt: [t]
   :END:

#+CAPTION:
#+NAME:   fig:ent1
#+ATTR_LATEX: :width \textwidth
[[file:./frequentFigs/entsd.png]]


** Detecting differential presence/absence
#+LaTeX: \vskip -6ex
*** Col left 							      :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.5
   :BEAMER_opt: [c]
   :END:

- Statistically compare traits to a pangenome

*** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.4
   :BEAMER_opt: [c]
   :END:

#+CAPTION:
#+NAME:   fig:scoary
#+ATTR_LATEX: :width \textwidth
file:./frequentFigs/scoary.png



* What does 10 years look like to /E. coli/?

** BoE Calculations for Doubling Time
High estimate:

0.013865 * 60 * 24 * 365 * 10
$\approx$ 72k generations

** BoE Calculations for Doubling Time
Medium estimate:
(5.9 / 2) * 365 * 10

$\approx$ 10k generations

Bååth 1998

(assuming generation time roughly equals half of turnover rate)

** Other estimates
No remaining bacteria after:
- 32 days
- 22 days
- 8 weeks
- 114 days

** Detection of Escherichia coli in sequenced soil

/E. coli/  approximately .092% prevelence in soil metagenomes



** Hypotheses
Stressed and outnumbered?

#+Beamer: \pause
*rapid* > incremental

#+BEAMER: \pause
$\dot{.\hspace{.095in}.}$ adaptation via mobile elements

* Mobile Genetic Elements

** Overview

#+BEAMER: \small
|                    |      *Kb* | *Mobility*  | *Detection*               |
|--------------------+-----------+-------------+---------------------------|
| Insertion Seq      |   .75-1.5 | transposase | ITR and transposase       |
| Non-composite Tn   |        <3 | IS          | IS with bonus genes       |
| Composite Tn       |      <5kb | double IS   | flanking ISs              |
| Integron           |       <10 | integrase*  | /attI/ site, AMR genes    |
| Genetic Island     |       >10 | phage       | GC Skew, ORF phylogeny    |
| Phage Inducible CI |      5-15 | phage       | phage inhibitors          |
| Prophage           | \approx50 | lysogeny    | integrase, tail, capsule  |
| Plasmid            |     1-1Mb | various     | run a gel,  /oriT/        |
|                    |           |             |                           |

** \url{https://github.com/nickp60/happpie/}
Horizontally Aquired Partial Pangenome of Inserted Elements
#+LaTeX: \vskip -2ex
#+LaTeX: \vspace{2em}
#+LaTeX: \vskip -6ex
#+CAPTION:
#+NAME:   fig:mobiled
#+ATTR_LATEX: :width \textwidth
file:./frequentFigs/mobiled.png



* In Closing
** Summary
- soil /E. coli/ is very diverse
- wide range of estimates exist for doubling time in soil
- diversiy prevents robust statistical trait association
- MGE phylogeny does not reflect genome phylogeny

** Future plans

- compare mobile pangenome of soil vs enteric /E. coli/
- source analysis: which /types/ of phages, etc
- incorporate genomic islands into analysis
- include regions interupted by IS, Tns, etc


** Sources
- https://www.nature.com/scitable/topicpage/transposons-the-jumping-genes-518
- https://www.nature.com/scitable/topicpage/transposons-the-jumping-genes-518
- https://www.researchgate.net/publication/283707425_The_Phage-Inducible_Chromosomal_Islands_A_Family_of_Highly_Evolved_Molecular_Parasites
- https://www.sciencedirect.com/science/article/pii/S0043135416302226
- http://www.bx.psu.edu/~ross/workmg/TranspositionCh9_files/
- https://www.frontiersin.org/articles/10.3389/fmicb.2018.00762/full

** Acknowledgments
\small
#+BEGIN_LaTeX
  \begin{columns}[onlytextwidth]
    \column{0.5\textwidth}
    \includegraphics[height=1cm]{2018-03-11_dc_figs/NUI_Galway_BrandMark_A_K.eps}\\
     NUIG Microbiology
      \begin{itemize}
        \item[\square] Dr. Fiona Brennan
        \item[\square] Dr. Florence Abram
        \item[\square] Soil and Environmental Microbiology Research Group
        \item[\square] Functional Environmental Microbiology Group
      \end{itemize}

    \column{0.5\textwidth}
    \vskip .25em
    \includegraphics[height=1cm]{2018-03-11_dc_figs/trimmed_jhi.png}\\
      James Hutton Institute, Dundee
      \begin{itemize}
        \item[\square] Dr. Leighton Pritchard
        \item[\square] Dr. Ashleigh Holmes
      \end{itemize}
\vskip 1cm
\pause
       \huge Questions?
  \end{columns}

#+END_LaTeX
