#+STARTUP: showall beamer
#+COLUMNS: %40ITEM %10BEAMER_env(Env) %9BEAMER_envargs(Env Args) %4BEAMER_col(Col) %10BEAMER_extra(Extra)
# +BEAMER_HEADER: \titlegraphic{\includegraphics[height=1.5cm]{InstLogo}}

#+TITLE: Genomic insights and virulence in soil-persistent /E. coli/
#+SUBTITLE: Version 1.0
#+AUTHOR: Nicholas Waters
#+DATE: \today
# #+SUBTITLE
#+INSTITUTE: Department of Microbiology \linebreak School of Natural Sciences \linebreak National University of Ireland, Galway}
#+LATEX_HEADER: \institute{Department of Microbiology\\ School of Natural Sciences\\ National University of Ireland, Galway}
#+LaTeX_HEADER:\usetheme[style=light]{NUIG}


#+OPTIONS: H:1 toc:nil

#+LaTeX_CLASS_OPTIONS: [17pt,aspectratio=169]
#+LATEX_HEADER: \renewcommand*\familydefault{\sfdefault}
#+LATEX_HEADER: \newcommand{\bt}{\textasciigrave}
#+LATEX_HEADER: \usepackage{xcolor}
#+LATEX_HEADER: \def \ttilde {\raisebox{-.6ex}\textasciitilde~}
#+LATEX_HEADER: \setlength\parindent{0pt} %set indent to zero
#+LATEX_HEADER: \setlength{\parskip}{1em}
#+LATEX_HEADER: \definecolor{bg}{HTML}{B1F4A0}
# +LATEX_HEADER: \lstset{basicstyle=\linespread{1.1}\ttfamily\scriptsize, breaklines=true, backgroundcolor=\color{bashcodebg}, xleftmargin=0.5cm, language=bash, showstringspaces=false, columns=fullflexible}
#+LATEX_HEADER: \usepackage{tcolorbox}
#+LATEX_HEADER: \usepackage{etoolbox}
#+LATEX_HEADER: \usepackage{geometry}
#+LATEX_HEADER: \usepackage[colorlinks = true, linkcolor = blue, urlcolor  = blue, citecolor = blue, anchorcolor = blue]{hyperref}
#+LATEX_HEADER: \let\oldv\verbatim
#+LATEX_HEADER: \let\oldendv\endverbatim
#+LATEX_HEADER: \def\verbatim{\par\setbox0\vbox\bgroup\scriptsize\oldv}
#+LATEX_HEADER: \def\endverbatim{\oldendv\egroup\fboxsep0pt \noindent\colorbox[gray]{0.8}{\usebox0}\par}
#+LaTeX_HEADER: \usepackage{array, booktabs, xcolor, tikz}


# +LaTeX_HEADER: \setbeamerfont{frametitle}{size=\normalsize}
#+LaTeX_HEADER: \usepackage{graphicx}
#+LaTeX_HEADER: \usetikzlibrary{arrows, calc, spy}


#+LaTeX_HEADER:\addtobeamertemplate{footnote}{\tiny}{} %\vspace{2ex}}


# light
#+BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon.pdf}}
# dark
# +BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon_D2.pdf}}


* A Brief History of Soil-persistent /E. coli/
#+BEGIN_LaTeX
%% handy timeline thing
\newcommand\ytl[2]{
\parbox[b]{4em}{\hfill{\color{headgrey}\bfseries\sffamily #1}~$\cdots$~}\makebox[0pt][c]{$\bullet$}\vrule\quad \parbox[c]{24em}{\vspace{7pt}\color{bonus}\raggedright\sffamily #2\\[7pt]}\\[-3pt]}
%
\begin{table}{\small
% \caption{A Brief Literature Review}
 \vskip -5mm
\centering
\begin{minipage}[t]{\linewidth}
\color{gray}
\rule{\linewidth}{1pt}
\ytl{1886}{Escherich: Discovery of \textit{E. coli}}
\ytl{1948}{Bardsley: Soil may act as reservoir for \textit{E. coli}}
\ytl{1963}{W. and J. Boyd: Cold persistence observed }
%\ytl{1967}{Klein, et al: Die-off related to metabolism rates}
%\ytl{1972}{Evans, et al: Drainage related to coliform counts} % and slurry spreading
\ytl{1988}{Fujioka and Shizumura: Alternative indicators suggested }
%\ytl{1992}{Tsai, et al: PCR detection of from soil}
% This date is wrong \ytl{1997}{Texier, et al: Stable populations exist in alpine grasslands}
\ytl{1995}{R. Sjogren: soil persistence across time and depth}
%\ytl{1998}{Byappanahalli and Fujioka: Soil extracts as growth media}
\ytl{2003}{Byappanahalli, et al: Soil persistence is widespread }
\ytl{2010}{Brennan, et al: Persistence in maritime temperate soils}
\bigskip
\rule{\linewidth}{1pt}%
\end{minipage}%
}
\end{table}
#+END_LaTeX


* Research Questions
- What types of /E. coli/ are able to persist in soil?
#+BEAMER: \pause
- What virulence factors are harboured by these strains?
#+BEAMER: \pause
- What can we infer about adaptation from these?
#+BEAMER: \pause
- Can we differentiate soil-persistent /E. coli/ from recent contamination?

* Workflow

#+CAPTION: workflow
#+NAME:   fig:workflow
#+ATTR_LATEX: :width .86\textwidth
[[file:./frequentFigs/workflow_v1_solid.png]]

* The data
- 171 isolates sequenced
- 22 failed assembly QC or ANI threshold
- *149 true /E. coli/ passed ANI screening*



* Sequence Typing
#+LaTeX: \vskip 1ex
** Col left                                                           :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.5
   :BEAMER_opt: [t]
   :END:
Clermont 2013
#+CAPTION:
#+NAME:   fig:phylo
#+ATTR_LATEX: :height .65\textheight
[[file:./frequentFigs/Phylogroups_solid.png]]

** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.5
   :BEAMER_opt: [t]
   :END:
Achman 7 gene MLST
#+CAPTION: Soil /E. Coli/ MLST
#+NAME:   fig:phylo
#+ATTR_LATEX: :height .65\textheight
[[file:./frequentFigs/MLST_plot.pdf]]


* Average Nucleotide Identity
http://widdowquinn.github.io/pyani/
#+BEGIN_LaTeX
\begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
  \node[anchor=south west,inner sep=0] (image) at (0,0){
  \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
\end{tikzpicture}
#+END_LaTeX


* Average Nucleotide Identity
http://widdowquinn.github.io/pyani/
#+BEGIN_LaTeX
\begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
  \node[anchor=south west,inner sep=0] (image) at (0,0){
  \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
    \spy on ($.53*(image.north)+0.61*(image.east)$) in node at ([xshift=4cm]image.east);
\end{tikzpicture}
#+END_LaTeX

* Virulence
- Search literature for genes implicated in virulence
- Select representative sequences for 65 virulence factors
- Use reciprocal translated blast to find occurrences
- Filter results, visualize

* Virulence Results
#+BEGIN_LaTeX
%\begin{tikzpicture}[remember picture, overlay]
%    \node[xshift=-5cm,yshift=-4.8cm] (innerimage) at (current page.north east){
\begin{tikzpicture}[spy using outlines={red,square,magnification=4, size=3.5cm,connect spies}]
    \node[anchor=south west,inner sep=0] (image) at (0,0){
\includegraphics[height=.85\paperheight]{./frequentFigs/20161122170535_blast_virulence_parser_output_heatmap_edited3.pdf}};
%        \begin{scope}[x={(image.south east)},y={(image.north west)}]
%        \foreach \x in {0,1,...,9} { \node [anchor=north] at (\x/10,0) {0.\x}; }
%        \foreach \y in {0,1,...,9} { \node [anchor=east] at (0,\y/10) {0.\y}; }
%        \end{scope}
%%%%%%%    \spy on ($0.55*(image.south east)+0.95*(image.north west)$) in node at ([yshift=1cm]image.north);
%\end{tikzpicture}};
\end{tikzpicture}
#+END_LaTeX


* Virulence Results
#+BEGIN_LaTeX
%\begin{tikzpicture}[remember picture, overlay]
%    \node[xshift=-5cm,yshift=-4.8cm] (innerimage) at (current page.north east){
\begin{tikzpicture}[spy using outlines={red,square,magnification=4, size=3.5cm,connect spies}]
    \node[anchor=south west,inner sep=0] (image) at (0,0){
\includegraphics[height=.85\paperheight]{./frequentFigs/20161122170535_blast_virulence_parser_output_heatmap_edited3.pdf}};
%        \begin{scope}[x={(image.south east)},y={(image.north west)}]
%        \foreach \x in {0,1,...,9} { \node [anchor=north] at (\x/10,0) {0.\x}; }
%        \foreach \y in {0,1,...,9} { \node [anchor=east] at (0,\y/10) {0.\y}; }
%        \end{scope}
    \spy on ($0.9*(image.south east)+0.19*(image.west)$) in node at ([xshift=4cm]image.east);
%%%%%%%    \spy on ($0.55*(image.south east)+0.95*(image.north west)$) in node at ([yshift=1cm]image.north);
%\end{tikzpicture}};
\end{tikzpicture}
#+END_LaTeX

* Enterobase comparison strains
#+LaTeX: \vskip 5ex
** Col left                                                           :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.45
   :END:
- One isolate from each Achman 7 MLST
- Total: 1185

** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.53
   :END:
#+CAPTION: workflow
#+NAME:   sources:
#+ATTR_LATEX: :width .9\textwidth
[[file:./frequentFigs/enterobase_source.pdf]]

* Pangenome Analysis

|            |    N | Core | Soft Core | Shell | Cloud  | total  |
|            |      | >99% |     95-99 | 15-95 | 0-15   |        |
|------------+------+------+-----------+-------+--------+--------|
| Soil       |  149 | 2662 |       505 |  2664 | 15,831 | 21,662 |
| Enterobase | 1193 | 1822 |       514 |  3346 | 73,606 | 79,288 |
| All        | 1342 | 1806 |       535 |  3406 | 78,121 | 83,868 |

* Assessing Assembly
#+CAPTION:
#+NAME:   fig:Annofilt
#+ATTR_LATEX: :height .6\paperheight
# +BEAMER \hspace*{-2em}
[[file:./frequentFigs/weird_gene3.png]]

* annofilt
** Col left                                                           :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.65
   :BEAMER_opt: [t]
   :END:
1. Create reference pangenome
2. Find genes next to contig borders
3. Blast against pangenome
4. Reject hits <  90% of CDS length

** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.33
   :BEAMER_opt: [t]
   :END:
#+BEAMER: \pause
#+CAPTION:
#+NAME:   fig:Annofilt
#+ATTR_LATEX: :width \textwidth
[[file:../annofilt/docs/icon/icon.pdf]]

* annofilt performance
https://nickp60.github.io/annofilt/
#+BEAMER: \vskip -5ex
** Col left                                                           :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.4
   :BEAMER_opt: [t]
   :END:
#+CAPTION:
#+NAME:   fig:Annofilt
#+ATTR_LATEX: :width \textwidth
[[file:./frequentFigs/ent2.png]]

** Col right                                                          :BMCOL:
   :PROPERTIES:
   :BEAMER_col: 0.4
   :BEAMER_opt: [t]
   :END:

#+CAPTION:
#+NAME:   fig:Annofilt
#+ATTR_LATEX: :width \textwidth
[[file:./frequentFigs/ents.png]]



* Interested in Genome Assembly?
Come ask about our tool *riboSeed* to assemble through rDNA repeats!
- Poster 466 zone D (upper gallery)
- Tuesday and Wednesday Evening
#+CAPTION: workflow
#+NAME:   fig:workflow
#+ATTR_LATEX: :width .6\textwidth
[[file:./frequentFigs/logo_1_dark.png]]


* Detecting differential presence/absence
[[file:./frequentFigs/scoary.png]]



* Future work
- subpangenomes
- virulence pathways
- AMR
- mobile elements

* Conclusions
- Soil /E. coli/ represent diverse lineages
- Soil /E. coli/ possess a range of virulence genes
- Pangenome analysis is very sensitive to annotation


# +BEAMER: \vskip 5ex \alert{\huge Questions?}

* Acknowledgments
\small
#+BEGIN_LaTeX
  \begin{columns}[onlytextwidth]
    \column{0.5\textwidth}
    \includegraphics[height=1cm]{2018-03-11_dc_figs/NUI_Galway_BrandMark_A_K.eps}\\
     NUIG Microbiology
      \begin{itemize}
        \item[\square] Dr. Fiona Brennan
        \item[\square] Dr. Florence Abram
        \item[\square] Soil and Environmental Microbiology Research Group
        \item[\square] Functional Environmental Microbiology Group
      \end{itemize}

    \column{0.5\textwidth}
    \vskip .25em
    \includegraphics[height=1cm]{2018-03-11_dc_figs/trimmed_jhi.png}\\
      James Hutton Institute, Dundee
      \begin{itemize}
        \item[\square] Dr. Leighton Pritchard
        \item[\square] Dr. Ashleigh Holmes
      \end{itemize}
\vskip 1cm
       \huge Questions?
  \end{columns}

#+END_LaTeX
