#+STARTUP: showall beamer
#+COLUMNS: %40ITEM %10BEAMER_env(Env) %9BEAMER_envargs(Env Args) %4BEAMER_col(Col) %10BEAMER_extra(Extra)
# +BEAMER_HEADER: \titlegraphic{\includegraphics[height=1.5cm]{InstLogo}}

#+TITLE: \small \newline\LARGE Soil-persistent /E. coli/ and Mobile elements
# +SUBTITLE: Or, an overview of mobile elements in /E. coli/
#+AUTHOR: Nicholas Waters
# +DATE: \today
# #+SUBTITLE
#+INSTITUTE: National University of Ireland, Galway, Ireland \linebreak The James Hutton Institute, Dundee, Scotland}
#+LATEX_HEADER: \institute{National University of Ireland, Galway, Ireland \\ The James Hutton Institute, Dundee, Scotland}

#+OPTIONS: H:2 toc:nil

#+LATEX_HEADER: \PassOptionsToPackage{draft}{graphicx}

# ################################################ Set the Aspect Ratio #############################
#+LaTeX_CLASS_OPTIONS: [16pt,aspectratio=169]
# +LaTeX_CLASS_OPTIONS: [15pt,aspectratio=43]

#+LATEX_HEADER: \renewcommand*\familydefault{\sfdefault} % make font sansserif
#+LATEX_HEADER: \newcommand{\bt}{\textasciigrave}
#+LATEX_HEADER: \usepackage{xcolor}
#+LATEX_HEADER: \def \ttilde {\raisebox{-.6ex}\textasciitilde~}
#+LATEX_HEADER: \setlength\parindent{0pt} %set indent to zero
#+LATEX_HEADER: \setlength{\parskip}{1em}
#+LATEX_HEADER: \definecolor{bg}{HTML}{B1F4A0}
# +LATEX_HEADER: \lstset{basicstyle=\linespread{1.1}\ttfamily\scriptsize, breaklines=true, backgroundcolor=\color{bashcodebg}, xleftmargin=0.5cm, language=bash, showstringspaces=false, columns=fullflexible}
#+LATEX_HEADER: \usepackage{tcolorbox}

#+LATEX_HEADER: \usepackage{etoolbox}
#+LATEX_HEADER: \usepackage{geometry}
#+LATEX_HEADER: \usepackage[colorlinks = true, linkcolor = blue, urlcolor  = blue, citecolor = blue, anchorcolor = blue]{hyperref}
#+LATEX_HEADER: \let\oldv\verbatim
#+LATEX_HEADER: \let\oldendv\endverbatim
#+LATEX_HEADER: \def\verbatim{\par\setbox0\vbox\bgroup\scriptsize\oldv}
#+LATEX_HEADER: \def\endverbatim{\oldendv\egroup\fboxsep0pt \noindent\colorbox[gray]{0.8}{\usebox0}\par}
#+LaTeX_HEADER: \usepackage{array, booktabs, xcolor, tikz}

#+LaTeX_HEADER: \usepackage{graphicx}
#+LaTeX_HEADER: \usepackage[]{algorithm2e}



#+LaTeX_HEADER: \usetikzlibrary{arrows, calc, spy, shapes}
#+LaTeX_HEADER: \tikzstyle{square} = [draw]
#+LaTeX_HEADER:\addtobeamertemplate{footnote}{\tiny}{} %\vspace{2ex}}

# set light/ dark theme here
#+LaTeX_HEADER:\usetheme[style=dark]{NUIG}
# light
# +BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon.pdf}}
# dark
#+BEAMER: \setbeamertemplate{itemize items}{\includegraphics[width=.6em, valign=c]{./frequentFigs/coli_icon_D2.pdf}}

* Background
# ** Sources
# #+BEAMER: \small
# - https://github.com/tseemann/prokka
# - https://github.com/nickp60/riboSeed
# - https://sanger-pathogens.github.io/Roary/


** Project Overview
- /E. coli/ has been found to persist stably in the soil
- Isolates were cultured from lysimeter leachate
- Strains were sequenced, resulting in *149* soil-persistent /E. coli/ genomes

** Research Questions
- What types of /E. coli/ are able to persist in soil?
#+BEAMER: \pause
- What virulence factors are harboured by these strains?
#+BEAMER: \pause
- *What can we infer about adaptation?*
#+BEAMER: \pause
- Can we differentiate soil-persistent /E. coli/ from recent contamination?

* /E. coli/ Pangenome
** Overview
Assembly  \Rightarrow Annotation \Rightarrow presense/absence matrix \newline
#+Beamer: \pause
riboSeed \Rightarrow Prokka\phantom{duck} \Rightarrow  Roary
#+Beamer: \pause

|            |    N | Core   | total  |
|------------+------+--------+--------|
| Soil       |  149 | 2662   | 21,662 |
| Enterobase | 1193 | *1822* | 79,288 |
| All        | 1342 | *1806* | 83,868 |


# ** Average Nucleotide Identity
# http://widdowquinn.github.io/pyani/
# #+BEGIN_LaTeX
# \begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
#   \node[anchor=south west,inner sep=0] (image) at (0,0){
#   \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
# \end{tikzpicture}
# #+END_LaTeX

# ** Average Nucleotide Identity
# :PROPERTIES:
# :BEAMER_opt: fragile
# :END:
# http://widdowquinn.github.io/pyani/
# #+BEGIN_LaTeX
# \begin{tikzpicture}[spy using outlines={black,square,magnification=5, size=6cm,connect spies}]
#   \node[anchor=south west,inner sep=0] (image) at (0,0){
#   \includegraphics[height=.75\paperheight]{../nrw_posters/utrecht2016/figs/ANIm_percentage_identity_edited.pdf}};
#     \spy on ($.53*(image.north)+0.61*(image.east)$) in node at ([xshift=4cm]image.east);
# \end{tikzpicture}
# #+END_LaTeX



# ** Annotation
# \begin{tikzpicture}%
#  \node [anchor=west] (note) at (-1,4) {\Large Partial};
#  \begin{scope}[xshift=1.5cm]
#      \node[anchor=south west,inner sep=0] (image) at (0,0) {\includegraphics[width=0.67\textwidth]{./frequentFigs/weird_gene3.png}};
#      \begin{scope}[x={(image.south east)},y={(image.north west)}]
#          \draw[red,ultra thick,rounded corners] (0.5,0.05) rectangle (0.55,0.15);
#          \draw [-latex, ultra thick, red] (note) to (0.48,0.1);
#     \end{scope}%
#  \end{scope}%
# \end{tikzpicture}



# ** Annotation Correction with annofilt performance
# # +LaTeX: \vskip -2ex
# # +LaTeX: \url{https://nickp60.github.io/annofilt/}\vspace{2em}
# # +LaTeX: \vskip -6ex
# *** Col left 							      :BMCOL:
#    :PROPERTIES:
#    :BEAMER_col: 0.4
#    :BEAMER_opt: [t]
#    :END:
# #+CAPTION:
# #+NAME:   fig:Annofilt
# #+ATTR_LATEX: :width .7\textwidth
# [[file:./frequentFigs/ent2d.png]]

# *** Col right                                                          :BMCOL:
#    :PROPERTIES:
#    :BEAMER_col: 0.4
#    :BEAMER_opt: [t]
#    :END:

# #+CAPTION:
# #+NAME:   fig:ent1
# #+ATTR_LATEX: :width .7\textwidth
# [[file:./frequentFigs/entsd.png]]


** Detecting differential presence/absence

- No gene(s) correlated with source
- /E. coli/ diversity adds noise


# #+LaTeX: \vskip -6ex
# *** Col left 							      :BMCOL:
#    :PROPERTIES:
#    :BEAMER_col: 0.5
#    :BEAMER_opt: [c]
#    :END:

# - Statistically compare traits to a pangenome

# *** Col right                                                          :BMCOL:
#    :PROPERTIES:
#    :BEAMER_col: 0.4
#    :BEAMER_opt: [c]
#    :END:

# #+CAPTION:
# #+NAME:   fig:scoary
# #+ATTR_LATEX: :width .8\textwidth
# file:./frequentFigs/scoary.png



* What does 10 years look like to /E. coli/ ?

** BoE Calculations for Doubling Time
- $\approx$ 72k generations
- $\approx$ 10k generations

** Other estimates
No viable coliforms after:
- 22 days
- 32 days
- 8 weeks
- 16 weeks

** /In situ/ Detection

/E. coli/  approximately .092% prevalence in soil metagenomes



** Hypothesis
Stressed and outnumbered?

#+Beamer: \pause
*rapid* > incremental

#+BEAMER: \pause
$\dot{.\hspace{.095in}.}$ adaptation via mobile elements

* Mobile Genetic Elements

** Overview

#+BEAMER: \small
|                    |      *Kb* | *Mobility*     | *Detection*              |
|--------------------+-----------+----------------+--------------------------|
| IS                 |   .75-1.5 | transposase    | ITR and transposase      |
| Non-composite Tn   |        <3 | IS             | IS with bonus genes      |
| Composite Tn       |      <5kb | double IS      | flanking ISs             |
| Integron           |       <10 | integrase      | /attI/ site, AMR genes   |
| Genetic Island     |       >10 | phage/plasmid  | GC Skew, ORF phylogeny   |
| Phage Inducible CI |      5-15 | phage          | phage inhibitors         |
| Prophage           | \approx50 | lysogeny       | integrase, tail, capsule |
| Plasmid            |     1-1Mb | various        | run a gel, /oriT/        |
|                    |           |                |                          |



** \url{https://github.com/nickp60/happie/}

Horizontally Aquired Partial Pangenome of Inserted Elements
1.  Run QC on genome assemmbly:
  - remove contigs < 800bp
2. Annotate whole genome (Prokka)
3. Filter out incomplete annotations with AnnoFilt
4. Detect mobile elements
  - Plasmids via mlplasmids
  - Prophages via ProphET
  - GIs via IslandPath-DIMOB
5. Extract and annotate mobile genome
6. Query both mobile and total genomes
  - detect AMR with resfinder
  - detect virulence genes with VFDB

** Genomic NJ on Pangenome
#+CAPTION:
#+NAME:   fig:genomed
#+ATTR_LATEX: :width .8\textwidth
file:./frequentFigs/genomicd.png


** Genomic NJ on Pan-mobilome
#+CAPTION:
#+NAME:   fig:mobiled
#+ATTR_LATEX: :width .8\textwidth
file:./frequentFigs/mobiled.png


* Comparing the pan-mobilome
** Overview
#+CAPTION:
#+NAME:   fig:overview
#+ATTR_LATEX: :width .6\textwidth
[[file:./2019-04-figs/mobile_sizes.png]]

** AMR
#+CAPTION:
#+NAME:   fig:amr
#+ATTR_LATEX: :width .6\textwidth
[[file:./2019-04-figs/perc_amr.png]]

# ** AMR validation
# #+CAPTION:
# #+NAME:   fig:amrvalid
# #+ATTR_LATEX: :width .7\textwidth
# [[file:./2019-04-figs/amr_ariba.png]]

** Enrichment Analysis (in progress)
- pyseer: kmers
- panX: diversity and visualization
- ML-based approaches

#+BEAMER: \pause
cold shock?


* In Closing
** Summary
What we know:
- soil /E. coli/ is very diverse
- wide range of estimates exist for doubling time in soil
- diversity prevents robust statistical trait association
- MGE phylogeny does not reflect genome phylogeny
- AMR genes may be be less frequently found in soil pan-mobilome

What we want to know:
- how the mobile pangenomes of soil vs enteric /E. coli/ compare
- which /types/ of phages, etc, are correlated by source
- which regions are interrupted by IS, Tns, etc


** Acknowledgments
\small
  \begin{columns}[onlytextwidth]
    \column{0.5\textwidth}
    \includegraphics[height=1cm]{2018-03-11_dc_figs/NUI_Galway_BrandMark_A_K.eps}\\
     NUIG Microbiology
      \begin{itemize}
        \item[\square] Dr. Fiona Brennan
        \item[\square] Dr. Florence Abram
        \item[\square] Soil and Environmental Microbiology Research Group
        \item[\square] Functional Environmental Microbiology Group
      \end{itemize}

    \column{0.5\textwidth}
    \vskip .25em
    \includegraphics[height=1cm]{2018-03-11_dc_figs/trimmed_jhi.png}\\
      James Hutton Institute, Dundee
      \begin{itemize}
        \item[\square] Dr. Leighton Pritchard
        \item[\square] Dr. Ashleigh Holmes
      \end{itemize}
\vskip 1cm
         \end{columns}
